content/qamp/qamp-projects.json

```diff
@@ -0,0 +1,1232 @@
+[
+  {
+    "cohort": {
+      "name": "QAMP Spring 23"
```

Comment:
Is there any chance that we can rename these to `Spring 2023`? or does the team want the `QAMP Spring 23` format?

Only asking because users will already be on the `QAMP` tab, and they understand what they're looking at. Having "QAMP" in all the filters, along with all the card tags, can equate to unnecessary noise, in my opinion.

Reply:
I like to proposal. It would keep the text simple.

Here we're only displaying the data that we get from the Airtable base. This means that the team in charge of updating that data is the one that could make this change. It would then be reflected here the next time the production build is generated.